source/jquery.flot.touchNavigate.js

```diff
@@ -152,15 +158,14 @@
 
         doubleTap = {
             recenterPlot: function(e) {
-                if (e && e.detail && e.detail.type === 'touchmove') {
-                    // do not recenter during touch moving;
-                    return;
+                if (e && e.detail && e.detail.type === 'touchstart') {
```

Comment:
Do recenter only for a double tap that trigger by touch start will make the most smooth interaction on a touch device. See gifs.
![touchend](https://user-images.githubusercontent.com/39820924/54022479-8eb77680-41cd-11e9-9154-8f462118c90b.gif)
![touchstart](https://user-images.githubusercontent.com/39820924/54022480-8f500d00-41cd-11e9-85ee-c981fcce673e.gif)